src/backend/joanie/edx_imports/tasks/certificates.py

```diff
@@ -177,18 +143,49 @@ def import_certificates_batch(start, stop, total, course_id, dry_run=False):
                         "name": organization.safe_translation_getter(
                             "title", language_code=language
                         ),
-                        "representative": signatory_name,
-                        "signature": signature,
                         "logo": image_to_base64(organization.logo)
                         if organization.logo
                         else None,
+                        "representative": None,
+                        "signature": None,
                     }
                 ],
             }
 
-        certificate_template = (
-            DEGREE if edx_certificate.mode == OPENEDX_MODE_VERIFIED else CERTIFICATE
-        )
+        has_signatory = edx_certificate.mode == OPENEDX_MODE_VERIFIED
+
+        if has_signatory:
```

Comment:
```suggestion
        if edx_certificate.mode == OPENEDX_MODE_VERIFIED:
```

Maybe I'm wrong but I don't see where `has_signatory` is reused